package.json

```diff
@@ -54,7 +54,7 @@
   },
   "devDependencies": {
     "@testing-library/jest-dom": "^5.12.0",
-    "@testing-library/react": "^11.2.7",
+    "@testing-library/react": "^12.0.0",
```

Comment:
Why are we updating dependencies here?

Reply:
My IDE will simply not behave, this goes for dependencies and linting errors. Will handle now!

Reply:
Much better! Can we also revert any changes to yarn.lock?

Reply:
> 
> 
> Much better! Can we also revert any changes to yarn.lock?


Resolved in 8205773